core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -508,7 +508,8 @@ class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boo
    * @throws KeeperException if there is an error while creating assignment
```

Comment:
On line 505, we should explain how this function is different from the one that returns PartitionReplicaAssignment.  Something like "this function does not return replicas which are being added or removed in a reassignment"

Reply:
Nothing in this class returns PartitionReplicaAssignment, are you referring to the parameters we accept?

Reply:
The function on line 505 (`createTopicAssignment`) doesn't return anything. It creates an assignment. I don't think it's worth distinguishing between reassignments here because we're creating a new topic

Reply:
yes, I agree, the JavaDoc for the `createTopicAssignment` functions is probably fine as-is.